places-solution-integrations/target/attach-data.md

```diff
@@ -0,0 +1,76 @@
+# Attaching Places Data to Target Events via Launch Rules
+
+This document assumes that you have Adobe Places implemented in your application. If you need help implementing Adobe Places, see [Places Extensions for AEP SDK](../../configure-places-in-the-sdk/README.md).
```

Comment:
Change to:
"This information assumes that you have Places implemented in your application. For more information about implementing Places, see..."